### DISCUSSION TITLE
Xamarin

### DISCUSSION THREAD

**Discussion by ImaginaryDevelopment** [snoopwpf/snoopwpf#247] [Ideas]
Can this be made to work on Xamarin too? This was a lifesaver working with WPF.

**Comment by batzen** [comment]
In theory that should be possible.
But i am not using Xamarin.
But if someone would like to contribute support for Xamarin i would be happy to discuss details about it.

I recently started to explore what i can achieve with Avalonia in one of my personal, non OSS, projects.
If i like Avalonia i would, most likely, make Snoop work with it.
That should also make it easier to support other UI frameworks, as Snoop is currently not really ready for that from an architectural standpoint as it misses some abstractions.
